BuildResidentialHPXML/measure.rb

```diff
@@ -951,12 +951,14 @@ def arguments(model)
     heating_system_type_choices << HPXML::HVACTypeFurnace
     heating_system_type_choices << HPXML::HVACTypeWallFurnace
     heating_system_type_choices << HPXML::HVACTypeFloorFurnace
-    heating_system_type_choices << HPXML::HVACTypeBoiler
+    heating_system_type_choices << "#{HPXML::HVACTypeBoiler}, In-Unit, w/ Baseboard"
```

Comment:
You're going to hate me but... in hindsight, it's probably simpler to just stick with `HPXML::HVACTypeBoiler` for the in-unit boilers (1st and 2nd heating systems).